docs-website-src/content/blockers.md

```diff
@@ -47,29 +52,40 @@ You can discover many of these issues by downloading `elevate-cpanel` and runnin
 * **cPanel is up to date**
   * You will need to be on a version mentioned in the "Latest cPanel & WHM Builds (All Architectures)" section at http://httpupdate.cpanel.net/
   * Mitigation: `/usr/local/cpanel/scripts/upcp`
-* **nameserver**
-  * cPanel provides support for a myriad of nameservers. (MyDNS, nsd, bind, powerdns). On RHEL 8 based distros, it is preferred that you always be on PowerDNS.
+* **Name Server**
+  * cPanel provides support for a myriad of name servers. (MyDNS, NSD, BIND, PowerDNS). On RHEL 8 based distributions, it is preferred that you always be on PowerDNS.
   * Mitigation: `/scripts/setupnameserver powerdns`
 * **MySQL**
-  * We will upgrade you automatically to MariaDB 10.6 during elevation if you do not do this in advance.
+  * If the version of MySQL/MariaDB installed on the system is not supported on RHEL 8 based distributions, you **must** upgrade to a supported version. If cPanel manages the MySQL installation, we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation.
+  * Elevation will block if a MySQL upgrade is in progress.
```

Comment:
Maybe move this to 'Conflicting Processes'?

Reply:
That section names scripts that, if running, are a conflict.  For a MySQL upgrade, is is something that cPanel/WHM just has running in the background.  So, I don't think it necessarily fits there.